recipes/pyntacle/meta.yaml

```
@@ -0,0 +1,57 @@
+{% set name = "pyntacle" %}
+{% set version = "1.3.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pyntacle-{{ version }}.tar.gz
+  sha256: 95f6aecc52794cc7a7eeb89773791c498b394874c01ade8a4ce85a0384e07513
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - pyntacle = pyntacle.pyntacle:App
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - numpy
+    - pip
+    - python >=3.6,<3.8
+  run:
+    - colorama ==0.4.4
+    - numba ==0.51.2
+    - ordered-set ==4.0.2
+    - pandas ==1.1.4
+    - psutil
+    - python >=3.6,<3.8
+    - python-igraph ==0.8.3
+    - seaborn ==0.11.0
+    - setuptools
+    - xlsxwriter ==1.3.7
+
+test:
+  imports:
+    - pyntacle
+    - pyntacle.algorithms
+  commands:
+    - pip check
+    - pyntacle --help
+  requires:
+    - pip
+
+about:
+  home: http://pyntacle.css-mendel.it
+  summary: A Python package for network analysis based on non-canonical metrics and HPC-Computing
+  dev_url: https://github.com/mazzalab/pyntacle
+  license: GPL-3.0-only
```

Looks like it [should be](https://github.com/mazzalab/pyntacle/blob/7d1d89c24262dcfa6cb2e4a68112a461fd020dc5/algorithms/global_topology.py#L16):
```suggestion
  license: GPL-3.0-or-later
```